modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -0,0 +1,52 @@
+[#anchor-and-file-names]
+= Anchor Names and File Names
+
+To optimize modular documentation, follow these guidelines for naming module anchors and files.
+
+Anchor names:: Provide an anchor for every module so that it can be identified when reused or cross-referenced. Give the anchor name the same or a similar name to the module heading, separated by underscores, and indicate the module type (`con`, `ref`, `proc`) at the end of the name. If applicable, consider also providing a module sub-type (`create`, `edit`, `delete`) before the module type:
```

Comment:
I'd say the sub-types are rather arbitrary in this case, and are not defined anywhere else. I'd advocate for either codifying them somehow (which means coming up with a list of sub-types that fits a reasonable number of use cases), or removing this part altogether.

My rationale is that this suggestion is a deviation from the rule _the filename should stay as close to the module title as possible_ without clearly formulated benefits. If we don't have a list of sub-types, this becomes just a random word tacked just before the end of the filename.

Reply:
You're right. I see your point here, with anchor names. I don't necessarily agree, however, with your argument for file names, i.e., that the file name too should match the module title as closely as possible. I've replied accordingly in your comments that follow.